lib/cusolver/linalg.jl

```diff
@@ -342,3 +342,14 @@ end
 
 LinearAlgebra.cholcopy(A::LinearAlgebra.RealHermSymComplexHerm{<:Any,<:CuArray}) =
     copyto!(similar(A, LinearAlgebra.choltype(A)), A)
+
+## inv
+
+function LinearAlgebra.inv(A::StridedCuMatrix{T}) where T <: BlasFloat
+    n = checksquare(A)
+    F = copy(A)
+    factors, ipiv, info = getrf!(F)
+    B = CuMatrix{T}(I, n, n)
```

Comment:
@maleadt Do you a better to create an identity matrix of `n` with CUDA.jl?

Reply:
This looks fine, and is specialized in GPUArrays. We could avoid a kernel to set the diagonal by using a 2D memcpy, but looking at the profiler most of the time is spent during the zero initialization, so there's not much to gain.